packages/token-cli/src/index.ts

```diff
@@ -0,0 +1,89 @@
+import { ensureFile, readFileSync, writeFile, existsSync } from 'fs-extra'
+import path from 'path'
+import yargs from 'yargs'
+import { createToken } from './createToken'
+import { FigmaResponse, getDesignToken } from './figma'
+import { mustBeDefined } from './utils'
+
+/**
+ * Figma
+ */
+const FIGMA_TOKEN = process.env.FIGMA_TOKEN
+const FIGMA_NODE_ID = process.env.FIGMA_NODE_ID
```

Comment:
FILE_ID?
https://www.figma.com/developers/api#get-local-variables-endpoint

Reply:
I fix it!